### PR TITLE
Remove feature group

### PR DESCRIPTION
**Pull request by saupchurch**
removes feature group per schema PR #672

### CODE REVIEW

**Comment by dcolligan** [comment]
these commits should be squashed

**Comment by saupchurch** [comment]
hopefully I did this right

File: ga4gh/client.py [hunk 1]
```diff
@@ -827,8 +799,7 @@ def __init__(self, backend):
             "individuals": self._backend.runGetIndividual,
             "rnaquantificationsets": self._backend.runGetRnaQuantificationSet,
             "rnaquantifications": self._backend.runGetRnaQuantification,
-            "expressionlevels": self._backend.runGetExpressionLevel,
-            "featuregroups": self._backend.runGetFeatureGroup
+            "expressionlevels": self._backend.runGetExpressionLevel
```
Comment (dcolligan) [inline ga4gh/client.py]: nitpick: it's good to keep the comma at the end of the list so only one line is changed in the diff, not 2 (and in a future diff where a line is added to this list for the same reason)

**Comment by dcolligan** [comment]
looks good to me

+1

**Comment by saupchurch** [comment]
ok - will resolve the nit and squash

**Comment by saupchurch** [comment]
squashed and updated

**Pull request merged**